dojo/tools/h1/parser.py

```diff
@@ -96,43 +114,60 @@ def get_findings(self, file, test):
-                    dynamic_finding=False,)
+                    dynamic_finding=False
+                )
                 finding.unsaved_endpoints = list()
                 dupes[dupe_key] = finding
         return dupes.values()
 
     def build_description(self, content):
-        date = content['attributes']['created_at']
-        date = datetime.strftime(datetime.strptime(date, "%Y-%m-%dT%H:%M:%S.%fZ"), "%Y-%m-%d")
-        reporter = content['relationships']['reporter']['data']['attributes']['username']
-        triaged_date = content['attributes']['triaged_at']
+        date = content["attributes"]["created_at"]
+        date = datetime.strftime(
+            datetime.strptime(date, "%Y-%m-%dT%H:%M:%S.%fZ"), "%Y-%m-%d"
+        )
+        reporter = content["relationships"]["reporter"]["data"]["attributes"][
+            "username"
+        ]
+        triaged_date = content["attributes"]["triaged_at"]
 
         # Build the description of the Dojo finding
-        description = "#" + content['attributes']['title']
+        description = "#" + content["attributes"]["title"]
         description += "\nSubmitted: {}\nBy: {}\n".format(date, reporter)
 
         # Add triaged date
         if triaged_date is not None:
             triaged_date = datetime.strftime(
-                datetime.strptime(triaged_date, "%Y-%m-%dT%H:%M:%S.%fZ"), "%Y-%m-%d")
+                datetime.strptime(triaged_date, "%Y-%m-%dT%H:%M:%S.%fZ"),
+                "%Y-%m-%d",
+            )
             description += "Triaged: {}\n".format(triaged_date)
 
         # Try to grab CVSS
         try:
-            cvss = content['relationships']['severity']['data']['attributes']['score']
+            cvss = content["relationships"]["severity"]["data"]["attributes"][
+                "score"
+            ]
             description += "CVSS: {}\n".format(cvss)
-        except:
+        except BaseException:
             pass
 
         # Build rest of description meat
-        description += "##Report: \n{}\n".format(content["attributes"]["vulnerability_information"])
+        description += "##Report: \n{}\n".format(
+            content["attributes"]["vulnerability_information"]
+        )
 
         # Try to grab weakness if it's there
         try:
-            weakness_title = content['relationships']['weakness']['data']['attributes']['name']
-            weakness_desc = content['relationships']['weakness']['data']['attributes']['description']
-            description += "\n##Weakness: {}\n{}".format(weakness_title, weakness_desc)
-        except:
+            weakness_title = content["relationships"]["weakness"]["data"][
+                "attributes"
+            ]["name"]
+            weakness_desc = content["relationships"]["weakness"]["data"][
+                "attributes"
+            ]["description"]
+            description += "\n##Weakness: {}\n{}".format(
+                weakness_title, weakness_desc
+            )
+        except BaseException:
```

Comment:
```suggestion
        except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50